src/js/constants.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by nicholas on 8/1/17.
+ */
+export const STUY_SPEC_API_URL = "http://rails-beanstalk-env.28enemwqsx" +
+  ".us-east-1.elasticbeanstalk.com"
```

Comment:
Meh, does it matter? https://stackoverflow.com/questions/32267017/performance-difference-between-a-regular-string-and-template-string